Cakefile

```diff
@@ -279,7 +279,10 @@ buildDocTests = (watch = no) ->
   outputFolder      = "docs/v#{majorVersion}"
 
   # Included in test.html
-  testHelpers = fs.readFileSync('test/support/helpers.coffee', 'utf-8').replace /exports\./g, '@'
+  readHelpersFile = (basename) ->
+    fs.readFileSync("test/support/#{basename}.coffee", 'utf-8').replace /exports\./g, '@'
+  testHelpers = readHelpersFile('helpers') + '\n'
+  testHelpers += readHelpersFile('abstract_syntax_tree_helpers')
```

Comment:
I'm not familiar with `doc:test`, the generated `docs/v2/test.html` looked visually correct (included the AST test helpers after the existing test helpers) but when I loaded that file in a browser I saw an `Uncaught SyntaxError: missing "` in the console